redirects-www.conf

```diff
@@ -250,6 +250,7 @@ rewrite ^/rad/(.*) https://transition.fec.gov/rad/$1 redirect;
 rewrite ^/pubrec/(.*) https://transition.fec.gov/pubrec/$1 redirect;
 rewrite ^/agenda/([0-9]+)/documents/(.*) /resources/updates/agendas/$1/$2 redirect;
 rewrite ^/agenda/([0-9]+)/(.*).pdf /resources/updates/agendas/$1/$2.pdf redirect;
+rewrite ^/agenda/agendas2002/(.*).pdf https://www.fec.gov/resources/updates/agendas/$1 redirect;
```

Comment:
@dorothyyeager The redirect path is really close and thanks for figuring this out! We'll want to do a few additional things. 

1. Update the redirect path to include the year's directory path and the extension. So it should look like this: 

`rewrite ^/agenda/agendas2002/(.*).pdf https://www.fec.gov/resources/updates/agendas/2002/$1.pdf redirect;`

I also checked to make sure the bucket contains those files, and they do. 

2. This will also be a problem for agendas from 2000, 2001, and 2003. So we'll need to handle those redirects too. Can you please add the following redirects?

```
rewrite ^/agenda/agendas2000/(.*).pdf https://www.fec.gov/resources/updates/agendas/2000/$1.pdf redirect;
rewrite ^/agenda/agendas2001/(.*).pdf https://www.fec.gov/resources/updates/agendas/2001/$1.pdf redirect;
rewrite ^/agenda/agendas2003/(.*).pdf https://www.fec.gov/resources/updates/agendas/2003/$1.pdf redirect;
```

Reply:
Done! Thank you so much @patphongs !!